### DISCUSSION TITLE
exercise trials

### DISCUSSION THREAD

**Discussion by mary-9555** [jspsych/jsPsych#2188] [Q&A] [answered]
Hi there.
I wanna know how i can define and code this:
If participant answered the first 3 trials correct, the main task begins. but if she didn't, exercise trials continue until 3 trials be correct overall (or in other way if 80% of trials be correct.). 
and i need **feedback** for these practice trials. 
Can i get an example please?

Thank you very much for your help in advance.

**Comment by jodeleeuw** [comment, marked as answer]
There's a video tutorial that includes a very similar structure. It's part of [Day 3 of Moving Research Online](https://www.youtube.com/watch?v=LP7o0iAALik). @becky-gilbert describes how to do this as part of the experiment she walks through. 

The relevant bit start around the 45 minute mark, but you may want to watch earlier bits to get the context.
